test/jdk/jdk/dynalink/TypeConverterFactoryMemoryLeakTest.java

```diff
@@ -25,6 +25,7 @@
  * @test
  * @bug 8198540
  * @summary Test TypeConverterFactory is not leaking method handles
+ * @run main/othervm -Xmx4M TypeConverterFactoryMemoryLeakTest
```

Comment:
I think `-Xmx4m` is risking it on some platforms that cannot go that low heap. Maybe do 128M, and bulk up the test allocations, so that GC definitely triggers?

Reply:
I removed the `-Xmx` option altogether on @plevart's suggestion to invoke `System.gc` explicitly instead. I added multiple @run directives, with all current GCs. (BTW, do I need to add `@requires vm.gc.Shenandoah` to the tests if I include Shenandoah?)

It's funny how ZGC and Shenandoah need 1 iteration less than all other GCs. If a test needs 12 iterations with most GCs, ZGC and Shenandoah need 11. If a test needs 2 iterations with others, these two will get it done in 1.